stubs/algopy-stubs/arc4.pyi

```diff
@@ -489,7 +489,9 @@ class Struct(metaclass=_StructMeta):
 
 class ARC4Client(typing.Protocol): ...
 
-_TABIResult_co = typing.TypeVar("_TABIResult_co", covariant=True)
+_TABIResult_co = typing.TypeVar(
+    "_TABIResult_co", bound=(algopy.BytesBacked | algopy.UInt64 | algopy.Bytes), covariant=True
```

Comment:
what about `bool`, `arc4.Struct`, others?

Reply:
Resolved this by removing the bounds completely